packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2518,7 +2520,6 @@ export class ContainerRuntime
 		let newState: boolean;
 
 		try {
-			this.submitIdAllocationOpIfNeeded(true);
```

Comment:
@Abe27342 - I was surprised to find this call _before_ the main ops were resubmitted.  Now this is being called afterwards.  Would you expect this to cause issues?

Reply:
Today I learned - This is a secondary call to cover all the IDs allocated the first time around, and needs to stay here.